step.yml

```diff
@@ -245,14 +234,13 @@ inputs:
         - "false"
         - "true"
   - bundletool_version: "0.13.4"
```

Comment:
Can't we remove this input now?

Reply:
Nope, it's still used for parsing information about the App Bundle. Check https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/blob/master/uploaders/aabuploader.go#L19.